app/controllers/nominations_controller.rb

```diff
@@ -1,7 +1,45 @@
 # frozen_string_literal: true
 
 class NominationsController < ApplicationController
```

Comment:
I think I'd like to split these into two controllers, one for requesting a nomination invite, and one for actually nominating. Thoughts?